serde/src/test/org/apache/hadoop/hive/serde2/TestJsonSerDe.java

```diff
@@ -74,6 +74,76 @@ public void testPrimativeDataTypes() throws Exception {
     Assert.assertEquals(Timestamp.ofEpochMilli(1549751270013L), results.get(4));
   }
 
+  @Test
```

Comment:
these unit tests are useful, thanks!
would you consider adding a qtest, which can describe the whole scenario better (create table with property, load data, having json in string column)?
I saw you added the sample json files under data/files, so I guess you wanted to add qtest

Reply:
Right, added nested_json_string.q